Formula/n/notation.rb

```diff
@@ -0,0 +1,32 @@
+class Notation < Formula
+  desc "CLI tool to sign and verify OCI artifacts and container images"
+  homepage "https://notaryproject.dev/"
+  url "https://github.com/notaryproject/notation.git",
+    tag:      "v1.0.0",
+    revision: "80e3fc4e2eeb43ac00bc888cf41101f5c56f1535"
+  license "Apache-2.0"
+  head "https://github.com/notaryproject/notation.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    project = "github.com/notaryproject/notation"
+    ldflags = %W[
+      -s -w
+      -X #{project}/internal/version.Version=v#{version}
+      -X #{project}/internal/version.GitCommit=#{Utils.git_short_head}
+      -X #{project}/internal/version.BuildMetadata=
```

Comment:
It is better to indicate that it is built by homebrew.
```suggestion
      -X #{project}/internal/version.BuildMetadata=Homebrew
```

Reply:
Sure! Adding.